spec/collections_publisher/publishing_parent_and_child_topic_spec.rb

```diff
@@ -48,21 +48,23 @@ def when_i_add_a_child_topic
   end
 
   def and_i_publish_it
-    click_link("Publish")
-    expect(page).to have_text("published")
+    click_on("Publish")
+    expect(page).to have_text(/published/i)
   end
 
   def then_i_can_view_both_on_gov_uk
     url = find_link(link)[:href]
     reload_url_until_status_code(url, 200)
 
-    click_link(link)
-    expect_rendering_application("collections")
-    expect_url_matches_live_gov_uk
-    expect(page).to have_content(child_title)
+    window = window_opened_by { click_link(link) }
```

Comment:
Is there a reason why this was needed?

Reply:
There was a reason, but i can't remember what it was failing on without it. Let me pull it out and see what it was

Reply:
I think it wasn't clicking the link without it

Reply:
Right i remember now. It's opening in a new tab, where as before it was loading in the same tab. We've agreed it loading in a new tab is fine.

Reply:
Sounds like something to include in the commit message ;)